tests/tox_env/python/virtual_env/package/test_python_package_util.py

```diff
@@ -51,6 +51,15 @@ def test_load_dependency_many_extra(pkg_with_extras: PathDistribution) -> None:
         assert str(left) == str(right)
 
 
+def test_load_dependency_poetry_many_extra() -> None:
+    requires = [Requirement('filelock<4.0.0,>=3.9.0; extra == "extras1" or extra == "extras2"')]
```

Comment:
instead of naming it poetry let's just say it `extra_or` :+1: And lets add an `extra_and` variant please?